<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="15,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="main_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="settings_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QWidget" name="serial_swidget" native="true">
           <property name="styleSheet">
            <string notr="true">#serial_swidget {
    border: 2px solid black;
}
</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>5</number>
            </property>
            <property name="verticalSpacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="serial_label">
              <property name="text">
               <string>串口配置</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="com_label">
              <property name="text">
               <string>端口</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="com_combobox"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="baud_label">
              <property name="text">
               <string>波特率</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="baud_combobox"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="databit_label">
              <property name="text">
               <string>数据位</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="databit_combobox"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="stopbit_label">
              <property name="text">
               <string>检验位</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="stopbit_combobox"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="checkbit_label">
              <property name="text">
               <string>停止位</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="checkbit_combobox"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="motion_label">
              <property name="text">
               <string>操作</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QPushButton" name="motion_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string>打开串口</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="rx_swidget" native="true">
           <property name="styleSheet">
            <string notr="true">#rx_swidget{
    border: 2px solid black;
}
</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="horizontalSpacing">
             <number>5</number>
            </property>
            <property name="verticalSpacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item row="2" column="0">
             <widget class="QRadioButton" name="rxascii_radio">
              <property name="text">
               <string>ASCII</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButton" name="rxhex_radio">
              <property name="text">
               <string>HEX</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="rxclear_button">
              <property name="text">
               <string>清空接收区</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="rxsave_button">
              <property name="text">
               <string>保存到文件</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="rx_label">
              <property name="text">
               <string>接收区设置</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="tx_swidget" native="true">
           <property name="styleSheet">
            <string notr="true">#tx_swidget {
    border: 2px solid black;
}
</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="horizontalSpacing">
             <number>5</number>
            </property>
            <property name="verticalSpacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item row="3" column="0">
             <widget class="QLabel" name="period_label">
              <property name="text">
               <string>自动发送周期</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QWidget" name="period_widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="period_lineEdit"/>
               </item>
               <item>
                <widget class="QLabel" name="ms_label">
                 <property name="text">
                  <string>ms</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QRadioButton" name="txascii_radio">
              <property name="text">
               <string>ASCII</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QRadioButton" name="txhex_radio">
              <property name="text">
               <string>HEX</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="newline_checkbox">
              <property name="text">
               <string>发送新行</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QCheckBox" name="autosend_checkbox">
              <property name="text">
               <string>自动发送</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="tx_label">
              <property name="text">
               <string>发送区设置</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="display_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="rx_textbrowser"/>
         </item>
         <item>
          <widget class="QWidget" name="tx_widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="4,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextEdit" name="tx_textedit"/>
            </item>
            <item>
             <widget class="QWidget" name="tx_sbwidget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>11</number>
               </property>
               <item>
                <widget class="QComboBox" name="txhistory_combobox">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="txload_button">
                 <property name="text">
                  <string>加载文件</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="txsend_button">
                 <property name="text">
                  <string>发送数据</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottom_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,2,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="welcome_label">
        <property name="text">
         <string>欢迎使用串口调试助手</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="txnum_label">
        <property name="text">
         <string>发送字节数：0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rxnum_label">
        <property name="text">
         <string>接收字节数：0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>清空计数</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
